index.html

```diff
@@ -0,0 +1,163 @@
+            <hr class="section_title_hr">
+          </div>
+        </div>
+        <ul class="row program d-flex flex-row justify-content-center">
+          <li class="col-12 col-md-2 pr-4 pl-0 mr-1">
+            <div class="program-block text-center row mb-5 p-4 bg-dark-light-custom">
+              <div class="col-sm-2 col-12 col-md-12 pr-0 pl-0">
+               <img src="assets/images/lecture.png" alt="lecture"/>
+              </div>
+              <h5 class="col-sm-5 col-12 col-md-12 pr-0 pl-0 orange section-subtitle">
+                Lecture
+              </h5>
+              <p class="custom_p_tag col-sm-5 col-12 col-md-12 pb-4 pr-0 pl-0 text-white">
+                Delivering seamless multiplayer experiences globally with Game Servers — with live Q&A
+              </p>
+            </div>
+          </li>
+          <li class="col-12 col-md-2 pr-4 pl-0 mr-1">
+            <div class="program-block text-center row mb-5 p-4 bg-dark-light-custom">
+              <div class="col-sm-2 col-12 col-md-12 pr-0 pl-0">
+                <img src="assets/images/CC.png" alt="cc"/>
+              </div>
+              <h5 class="col-sm-5 col-12 col-md-12 pr-0 pl-0 orange section-subtitle">
+                CC Exhibition
+              </h5>
+              <p class="custom_p_tag col-sm-5 col-12 col-md-12 pb-4 pr-0 pl-0 text-white">
+                Connecting with gaming creators: Four things developers should know — with live Q&A
+              </p>
+            </div>
+          </li>
+          <li class="col-12 col-md-2 pr-4 pl-0 mr-1">
+            <div class="program-block text-center row mb-5 p-4 bg-dark-light-custom">
+              <div class="col-sm-2 col-12 col-md-12 pr-0 pl-0">
+                <img src="assets/images/forum.png" alt="forum"/>
+              </div>
+              <h5 class="col-sm-5 col-12 col-md-12 pr-0 pl-0 orange section-subtitle">
+                Forum
+              </h5>
+              <p class="custom_p_tag col-sm-5 col-12 col-md-12 pb-4 pr-0 pl-0 text-white">
+                Cloud Native Databases and Storage for Games and optimization techniques on framerates — with live Q&A
+              </p>
+            </div>
+          </li>
+          <li class="col-12 col-md-2 pr-4 pl-0 mr-1">
+            <div class="program-block text-center row mb-5 p-4 bg-dark-light-custom">
+              <div class="col-sm-2 col-12 col-md-12 pr-0 pl-0">
+                <img src="assets/images/workshop.png" alt="workshop"/>
+              </div>
+              <h5 class="col-sm-5 col-12 col-md-12 pr-0 pl-0 orange section-subtitle">
+                Workshop
+              </h5>
+              <p class="custom_p_tag col-sm-5 col-12 col-md-12 pr-0 pb-3 pl-0 text-white">
+                Less time porting and certing, more time creating fast and reliable environmental setup — with live Q&A
+              </p>
+            </div>
+          </li>
+        </ul>
```

Comment:
- [ ] **Required:** This section should contain 5 articles.
![image](https://user-images.githubusercontent.com/27709832/152189729-ef46763c-94f9-4def-9cff-d2f37e69d0dd.png)

- [ ] **Required:** Consider replacing the `div`s with `article`s to make more semantic tags.
- [ ] **Required:** The articles should be aligned [card decks](https://getbootstrap.com/docs/4.0/components/card/#card-decks) using rather than the current [masonary cards](https://getbootstrap.com/docs/4.0/components/card/#card-columns) nature

Your Implementation | Behance Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/152191030-f13167d1-bd8d-41b9-a5a2-c5ccdc139c0a.png) | ![image](https://user-images.githubusercontent.com/27709832/152191231-41dc39bc-b26e-447c-8df8-f955b3e0463c.png)